man/git-extras.md

```diff
@@ -31,6 +31,8 @@ git-extras(1) -- Awesome GIT utilities
    - **git-chore(1)** Create chore branch
    - **git-clear-soft(1)** Soft clean up a repository
    - **git-clear(1)** Rigorously clean up a repository
+   - **<NAME>(1)** <short description>
+   - **git-coauthor(1)** Add a co-author to the last commit
```

Comment:
Now I know why the `git-coauthor` is not added automatically.
The `**<NAME>(1)** <short description>` is added instead because the placeholder is not updated.

Reply:
Ahhhh. Thanks, I will make a fix 👍